docs/strucpp-migration/03-arduino-runtime-adaptation.md

```
@@ -0,0 +1,316 @@
+always uses `Config0` as the configuration name -- this is not user-configurable).
+
+## Step 3.1: Create New Arduino Sketch
+
+**New file**: `resources/sources/Baremetal/Baremetal.ino`
+
+This is a **static** sketch -- the same code for every project. It dynamically discovers
+the project structure from STruC++ runtime types at `setup()` time.
+
+### Sketch Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)